src/containers/CustomNetworkHome/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { KeyboardEvent, MouseEvent, useState } from 'react'
+import { KeyboardEvent, MouseEvent as ReactMouseEvent, useState } from 'react'
```

Comment:
Does importing it as `MouseEvent` not work?

Reply:
No because we also need the native MouseEvent type in the file.